libvast/src/system/accountant.cpp

```diff
@@ -236,8 +240,8 @@ struct accountant_state_impl {
                     cfg.uds_sink.path, s.error());
         }
       } else {
-        auto s
-          = detail::make_output_stream(cfg.uds_sink.path, cfg.uds_sink.type);
+        auto s = detail::make_output_stream(root / cfg.uds_sink.path,
```

Comment:
The root should only be prepended when the `sink_path` is relative.

Reply:
That's a no-op. `a / b == b` if `b` is absolute.

Reply:
See https://godbolt.org/z/qzWbW7z63 for a minimal example.

Reply:
Surprising, but ok.

Reply:
How is that surprising? What do you expect to happen when you append an absolute path?